apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common.Tests/Trace/SimpleManagedTracerTest.cs

```diff
@@ -79,6 +84,49 @@ private static bool Is1MsSpan(TraceSpan span)
             return duration.Seconds == 0 && duration.Nanos == 1_000_000;
         }
 
+        /// <summary>
+        /// Used to ensure that <see cref="Timestamp"/>s manipulated by
+        /// <see cref="SimpleManagedTracer"/> are normalized after
+        /// manipulation.
+        /// </summary>
+        private static bool IsNormalized(Timestamp timestamp) =>
+            timestamp.Nanos >= 0 &&
+            timestamp.Nanos <= MaxNanos &&
```

Comment:
I think I'd just use Duration.NanosecondsPerSecond and make the range exclusive.

Reply:
Done